src/webview/mod.rs

```diff
@@ -1140,8 +1140,27 @@ pub trait WebviewExtWindows {
   /// Returns WebView2 Controller
   fn controller(&self) -> ICoreWebView2Controller;
 
-  // Changes the webview2 theme.
+  /// Changes the webview2 theme.
   fn set_theme(&self, theme: Theme);
+
+  /// Sets [the memory usage target level][1] to 'Low'. There are two levels 'Low' and 'Normal' and
+  /// the default level is 'Normal'. When the application is going inactive, setting the level to
+  /// 'Low' can significantly save the memory used by the application.
+  ///
+  /// Passing `true` to `enabled` argument means the WebView will switch the level to 'Low' and
+  /// passing `false` will switch to 'Normal'.
+  ///
+  /// The logic when application goes to inactive/active is specific to the application.
+  /// [`tao::event::WindowEvent::Focused`] event may be useful.
```

Comment:
```suggestion
  /// When to best use this mode depends on the app in question. Most commonly it's called when
  /// the app's visiblity state changes.
```
I'd rather recommend using it when the app goes invisible, but if you prefer we can include both, visibility and focus changes.